.eslintrc.json

```diff
@@ -13,10 +13,5 @@
         "ecmaVersion": 2018
     },
     "rules": {
-        "no-trailing-spaces": "error",
-        "no-empty": 1,
-        "no-console": 1,
-        "no-irregular-whitespace": 1,
-        "no-unused-vars":1
```

Comment:
What are the reasons to remove all these rules?

Reply:
These are not the default rules, no other reason